### ISSUE TITLE
Add an icon when media has descriptions

### ISSUE THREAD

**Issue by lawremipsum** [florence-social/mastodon-fork#76]
See this PR on MSP Social: https://github.com/lawremipsum/mspsocial-mastodon/pull/14

This patch has been in place for 10 months on MSP Social and serves two purposes: makes users more aware of image descriptions, thereby promoting their use, and provides an unobtrusive way to see at a glance whether or not an image has a description. This is especially helpful on mobile, where it is a lot of work to check to see if an image has a description—it's helpful to be able to see right away whether it's even worth the effort to look.

Ideally, the icon would probably be a button that would reveal descriptions, but that UI hasn't been implemented.

This code is written and ready to go, I just don't know how to easily separate it from the rest of my fork to submit it as a pull request. I might eventually figure it out, but hopefully someone more adept at git can make it happen faster.

**Comment by ghost** [comment]
https://git.multiple-creature.party/monsterpit/mastodon also implements just Always Showing Media Descriptions and we would love to see that as an option.

also, we'd make a small protest that mastodon is way loaded with icons and a textual button would be more clear (see also the problem everyone had with not knowing what the "expand all content warnings in this thread" button did)

**Comment by ryliejamesthomas** [comment]
@Fidgetcetera Yeah, I use Subway Tooter to access Fedi and it just always shows the descriptions and I really like that feature. Sometimes I put extra info in there and forget that Masto hides them.
